package.json

```diff
@@ -7,16 +7,11 @@
   "main": "src/index.js",
   "types": "src/index.d.ts",
   "scripts": {
-    "lint": "aegir lint",
-    "build": "aegir build && cp ./src/index.d.ts ./dist",
-    "test": "yarn test:node",
-    "test:node": "aegir test --target node",
-    "test:browser": "aegir test --target browser",
-    "release": "aegir release",
-    "release-minor": "aegir release --type minor",
-    "release-major": "aegir release --type major",
-    "coverage": "aegir coverage",
-    "coverage-publish": "aegir coverage --provider coveralls"
+    "lint": "eslint . --ext js,json",
+    "lint:fix": "eslint . --ext js,json --fix",
+    "build": "./node_modules/.bin/babel src --out-dir dist",
```

Comment:
```suggestion
    "build": "babel src --out-dir dist",
```

Reply:
I reckoned there had to be a better way. I don't understand why the version of the babel docs I found didn't suggest this